app/lib/common/widgets/attachments/attachment_item.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show Attachment, FfiBufferUint8;
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+// Attachment item UI
+class AttachmentItem extends ConsumerWidget {
+  final Attachment attachment;
+  const AttachmentItem({super.key, required this.attachment});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    var msgContent = attachment.msgContent();
+    String? mimeType = msgContent.mimetype();
```

Comment:
attachments are strongly typed (`attachment.typeStr()`) why are we doing a mime-type-guessing-game here? Even if it is an image, if it was uploaded as a `file` to the network, we should just consider it a file ... not try to guess it to be something else...

Reply:
I guess I wasn't aware of it. Or maybe I inferred that wrong, will use this determination.

Reply:
Fixed by c6ea65a.

Reply:
Fixed by ac0da96